### PR TITLE
Bread cats

### PR DESCRIPTION
**Pull request by aboutsema-aa**
Just that

breadcat

### CODE REVIEW

**Comment by friedger** [comment]
Could you please provide an animated gif?

**Comment by rock3r** [comment]
Yup, gif please. See [here](https://github.com/novoda/droidcon-booth#how-to-take-part-) for instructions

**Comment by aboutsema-aa** [comment]
Yea sorry, had to leave for Chet ;)

I've added a gif just now. Enjoy

**Comment by ouchadam** [comment]
this also drops our readme! need to revert those changes :ghost:

**Comment by aboutsema-aa** [comment]
Check
On Oct 30, 2015 10:29, "Adam Brown" <notifications@github.com> wrote:

> this also drops our readme! need to revert those changes [image: :ghost:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/droidcon-booth/pull/47#issuecomment-152487165>.
>

-- 
*This email may contain or reference confidential information and is 
intended only for the individual to whom it is addressed.  Please refrain 
from distributing, disclosing or copying this email and the information 
contained within unless you are the intended recipient.  If you received 
this email in error, please notify us at legal@appannie.com 
<legal@appannie.com>** immediately and remove it from your system.*

**Comment by ryanbateman** [comment]
LGTM!

File: mobile/src/androidTest/java/com/novoda/canvas/BreadCatTest.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        R.drawable.cat4,
+        R.drawable.cat5,
+        R.drawable.cat6
+    };
+
+    // Config
+    final static int ANIMATION_DURATION = 100;
+    final static int STILL_DURATION = 500;
+
+    // Objects
```
Comment (blundell) [inline mobile/src/androidTest/java/com/novoda/canvas/BreadCatTest.java:28]: `int currentCat` is an object naow?

File: mobile/src/androidTest/java/com/novoda/canvas/BreadCatTest.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        getParent().addView(imageBottom);
+        getParent().addView(imageTop);
+
+        lockAndLoad();
+    }
+
+    private void setNewCat() {
+        for(int i = 0; i < 10; i ++) {
+            int randomCat = getRandomCat();
+            if(randomCat != currentCat) {
```
Comment (blundell) [inline mobile/src/androidTest/java/com/novoda/canvas/BreadCatTest.java:59]: interesting check :) the probability is low right and even if it did equal it then resetting the cat isn't that big a change :dancer:

**Pull request merged**